cmd/util/ledger/migrations/cadence_values_migration.go

```diff
@@ -317,26 +419,30 @@ func (t *cadenceValueMigrationReporter) Migrated(
 
 func (t *cadenceValueMigrationReporter) Error(err error) {
 
-	message := t.errorMessageHandler.FormatError(err)
-
 	var migrationErr migrations.StorageMigrationError
 
-	if errors.As(err, &migrationErr) {
-		storageKey := migrationErr.StorageKey
-		storageMapKey := migrationErr.StorageMapKey
-		migration := migrationErr.Migration
-
-		t.log.Error().Msgf(
-			"failed to run %s in account %s, domain %s, key %s: %s",
-			migration,
-			storageKey.Address,
-			storageKey.Key,
-			storageMapKey,
-			message,
-		)
-	} else {
-		t.log.Error().Msgf("failed to run migration: %s", message)
+	if !errors.As(err, &migrationErr) {
+		panic(cadenceErrors.NewUnreachableError())
```

Comment:
Is it guaranteed to always report a `StorageMigrationError`?
If so, maybe we could change the signature of `Error` function to only accept a `StorageMigrationError`

Reply:
agreed, we might consider just changing the the type 👍